test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
This isn't an official designation or anything, just a triple @timbl recommended to test the serialization base in the output.